scripts/rf_update

```diff
@@ -12,9 +12,54 @@ Brief : This script uses the redfish_utilities module to perform updates
 """
 
 import argparse
+import os
+import re
+import shutil
+import socket
+import sys
+import threading
+import time
+
 import redfish
 import redfish_utilities
 
+if sys.version_info > ( 3, ):
+    import http.server
+else:
+    import SimpleHTTPServer
+    import SocketServer
+
+WEB_SERVER_PORT = 8888
```

Comment:
Future enhancement: Might want to perform try/catch statements to probe for a good port incase 8888 is in use.